lib/pipedrive/base.rb

```diff
@@ -14,14 +14,7 @@ def make_api_call(*args)
       method = args[0]
       fail 'method param missing' unless method.present?
       url = build_url(args, params.delete(:fields_to_select))
-      begin
-        res = connection.__send__(method.to_sym, url, params)
-      rescue Errno::ETIMEDOUT
-        retry
-      rescue Faraday::ParsingError
```

Comment:
Why you have removed? Its means that code will be retry till it connected.

Reply:
@simonoff I confirm that this is an issue, when you hit the rate limit you get the cloudflare page and the gem keeps the connection open with retries. (ref: https://pipedrive.readme.io/docs/core-api-concepts-rate-limiting)
Can we please merge this?